guides/doc-Managing_Configurations_Puppet/docinfo.xml

```diff
@@ -0,0 +1,14 @@
+<title>Managing Configurations Using Puppet Integration in Satellite</title>
+<productname>Red Hat Satellite</productname>
+<productnumber>7.0</productnumber>
+<subtitle>Using Puppet integration to manage configurations of hosts with Puppet modules and reporting to Satellite</subtitle>
```

Comment:
This is a tiny bit confusing to parse, (although I'm not super familiar with Satellite).
The 'with' is problematic for translation because it's not clear if you use Puppet modules to manage configurations, or you manage configurations that have Puppet modules. I think it's the latter but a machine translator probably wouldn't know which.

The ending 'and reporting to satellite' seems tacked on and the verb is in a different tense (present continuous instead of infinitive like the previous verb). 

This suggestion might not be accurate so take with a pinch of salt:

`Using Puppet integration to manage and report configurations and <and_whatever_is_being_reported> of hosts that have Puppet modules.`

If that suggestion is no good, I still suggest to make it clearer based on the issues I mention here.